.circleci/config.yml

```diff
@@ -272,7 +272,7 @@ steps_bootstrap: &steps_bootstrap
     - *attach_workspace
     - *run_compiler_version
     - run: cd boost-root && ./bootstrap.sh -with-toolset=$TOOLSET || cat ./bootstrap.log
-    - run: cd boost-root && ./b2 headers > /dev/null
+    - run: cd boost-root && ./b2 -j2 headers > /dev/null
```

Comment:
First, this PR is mixing too many changes, include changes which **already exist** in the `develop` branch, e.g.

https://github.com/boostorg/gil/blob/dfbbec153131ac4d9f6c3d1b59b3dd2389695e43/.circleci/config.yml#L275

Second, the source/target of this PR does not look right to me:

![image](https://user-images.githubusercontent.com/80741/61177512-b5504880-a5d7-11e9-9228-e2338bb02570.png)

The GIL upstream only accepts PRs based on `develop`, but this info presented above does not make it clear to understand what is this PR based on. 

That, combined with the huge number of changes makes this PR hardly reviewable or acceptable. Sorry.